publish.sh

```diff
@@ -39,6 +40,9 @@ publish lalrpop-util
 publish lalrpop-snap
 publish lalrpop
 
+git tag $VERSION
+git push origin $VERSION
```

Comment:
I think this should be [`git push origin tag $VERSION`](https://git-scm.com/docs/git-push#git-push-ltrefspecgt82308203). Alternatively you can `git push origin --tags`, but that will push *all* tags, which might not be desirable depending on your local repository state.

Reply:
I think pushing only one is preferred. Added the `tag` option.